packages/core/src/modules/slack/use-cases/send-stackbot-mass-notification.ts

```diff
@@ -0,0 +1,18 @@
+import { job } from '@/infrastructure/bull/use-cases/job';
+import { db } from '@/infrastructure/database';
+
+export async function sendStackBotMassNotification(message: string) {
```

Comment:
Changed the parameter from `data` from a Bull job to just a string. Also, I believe this should still support rich text messages so long as the message is passed into this function with the correct formatting.